CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration_UI.uc

```diff
@@ -240,7 +240,7 @@ static protected function EventListenerReturn CovertActionCompleted(Object Event
 
 		HQPres = `HQPRES;
 		HQPres.NotifyBanner(default.strInfiltrationReady, MissionState.GetUIButtonIcon(), MissionState.GetMissionObjectiveText(), default.strCanWaitForBonusOrLaunch, eUIState_Good);
-		HQPres.PlayUISound(eSUISound_SoldierPromotion);
+		`XSTRATEGYSOUNDMGR.PlaySoundEvent(Activity.GetMyTemplate().CompleteSound);
```

Comment:
This isn't the CA being completed per say, it's the UI notification for transition from CA to infil mission site.

So `CompleteSound` is at least badly named in this case

Reply:
I'll change it to MissionReadySound